src/control/Control.Zoom.js

```diff
@@ -27,25 +28,25 @@ export const Zoom = Control.extend({
 
 		// @option zoomInTitle: String = 'Zoom in'
 		// The title set on the 'zoom in' button.
-		zoomInTitle: 'Zoom in',
+		zoomInTitle: '',
 
 		// @option zoomOutText: String = '<span aria-hidden="true">&#x2212;</span>'
 		// The text set on the 'zoom out' button.
 		zoomOutText: '<span aria-hidden="true">&#x2212;</span>',
 
 		// @option zoomOutTitle: String = 'Zoom out'
 		// The title set on the 'zoom out' button.
-		zoomOutTitle: 'Zoom out'
+		zoomOutTitle: ''

```

Comment:
Should we remove the options completely?

Reply:
Sounds good to me! But then letting the doc string there or is there another way to document a non declared option ?